src/base/net/geoipmanager.cpp

```diff
@@ -45,7 +45,7 @@
 static const char DATABASE_URL[] = "https://geolite.maxmind.com/download/geoip/database/GeoLite2-Country.mmdb.gz";
 static const char GEOIP_FOLDER[] = "GeoIP";
 static const char GEOIP_FILENAME[] = "GeoLite2-Country.mmdb";
-static const int CACHE_SIZE = 1000;
+//static const int CACHE_SIZE = 1000;
```

Comment:
I just wanted to point out that with a VCS commented out statements are nearly useless.